app/views/results/index.html.erb

```diff
@@ -1,6 +1,11 @@
 <%= content_for :page_title, "#{@number_of_courses_string}" %>
 
-<% if @results_view.provider_filter? %>
+<% if FeatureFlag.active?(:new_filters) %>
+  <h1 class="govuk-heading-l" data-qa="heading">
+    <span class="govuk-caption-l">Teacher training courses</span>
+    <%= "#{@results_view.number_of_courses_string} found" %>
```

Comment:
just making a note that this has been removed? I might find out why in a later commit 👍

Reply:
Sorry that was an error. Fixed now